src/tracer.js

```diff
@@ -107,8 +84,6 @@ export const isCallbacked = handlerReturnValue => {
 export const endTrace = async (functionSpan, handlerReturnValue) => {
   try {
     if (functionSpan && !isSwitchedOff() && isAwsEnvironment()) {
-      // eslint-disable-next-line no-undef
-      TracerGlobals.timeoutTimer && clearTimeout(TracerGlobals.timeoutTimer);
```

Comment:
Shouldn't you remove also `clearTimeout`?

Reply:
There is not TracerGlobals.timeoutTimer any more